erpnext/accounts/doctype/cash_flow_mapping_template_details/cash_flow_mapping_template_details.json

```
@@ -1,94 +1,33 @@
 {
- "allow_copy": 0, 
- "allow_guest_to_view": 0, 
- "allow_import": 0, 
- "allow_rename": 0, 
- "autoname": "field:mapping", 
- "beta": 0, 
- "creation": "2018-02-08 10:18:48.513608", 
- "custom": 0, 
- "docstatus": 0, 
- "doctype": "DocType", 
- "document_type": "", 
- "editable_grid": 1, 
- "engine": "InnoDB", 
+ "actions": [],
+ "creation": "2022-02-11 11:25:05.336846",
```

```suggestion
 "creation": "2018-02-08 10:18:48.513608",
```

We should get this Frappe bug fixed 🥲